scripts/us_census/pep/annual_population/download.sh

```diff
@@ -0,0 +1,82 @@
+        https://www2.census.gov/programs-surveys/popest/tables/2000-2010/intercensal/county/co-est00int-01-36.csv
+        https://www2.census.gov/programs-surveys/popest/tables/2000-2010/intercensal/county/co-est00int-01-37.csv
+        https://www2.census.gov/programs-surveys/popest/tables/2000-2010/intercensal/county/co-est00int-01-38.csv
+        https://www2.census.gov/programs-surveys/popest/tables/2000-2010/intercensal/county/co-est00int-01-39.csv
+        https://www2.census.gov/programs-surveys/popest/tables/2000-2010/intercensal/county/co-est00int-01-40.csv
+        https://www2.census.gov/programs-surveys/popest/tables/2000-2010/intercensal/county/co-est00int-01-41.csv
+        https://www2.census.gov/programs-surveys/popest/tables/2000-2010/intercensal/county/co-est00int-01-42.csv
+        https://www2.census.gov/programs-surveys/popest/tables/2000-2010/intercensal/county/co-est00int-01-44.csv
+        https://www2.census.gov/programs-surveys/popest/tables/2000-2010/intercensal/county/co-est00int-01-45.csv
+        https://www2.census.gov/programs-surveys/popest/tables/2000-2010/intercensal/county/co-est00int-01-46.csv
+        https://www2.census.gov/programs-surveys/popest/tables/2000-2010/intercensal/county/co-est00int-01-47.csv
+        https://www2.census.gov/programs-surveys/popest/tables/2000-2010/intercensal/county/co-est00int-01-48.csv
+        https://www2.census.gov/programs-surveys/popest/tables/2000-2010/intercensal/county/co-est00int-01-49.csv
+        https://www2.census.gov/programs-surveys/popest/tables/2000-2010/intercensal/county/co-est00int-01-50.csv
+        https://www2.census.gov/programs-surveys/popest/tables/2000-2010/intercensal/county/co-est00int-01-51.csv
+        https://www2.census.gov/programs-surveys/popest/tables/2000-2010/intercensal/county/co-est00int-01-53.csv
+        https://www2.census.gov/programs-surveys/popest/tables/2000-2010/intercensal/county/co-est00int-01-54.csv
+        https://www2.census.gov/programs-surveys/popest/tables/2000-2010/intercensal/county/co-est00int-01-55.csv
+        https://www2.census.gov/programs-surveys/popest/tables/2000-2010/intercensal/county/co-est00int-01-56.csv $urls"
+urls="https://www2.census.gov/programs-surveys/popest/datasets/2010-2020/counties/totals/co-est2020.csv $urls"
+urls="https://www2.census.gov/programs-surveys/popest/tables/1990-2000/cities/totals/su-99-7_us.txt $urls"
+urls="https://www2.census.gov/programs-surveys/popest/datasets/2000-2010/intercensal/cities/sub-est2010-alt.csv $urls"
+urls="https://www2.census.gov/programs-surveys/popest/datasets/2010-2019/cities/totals/sub-est2019_all.csv $urls"
+
+urls="https://www2.census.gov/programs-surveys/popest/tables/1980-1990/state/asrh/st7080ts.txt"
+urls="https://www2.census.gov/programs-surveys/popest/tables/1980-1990/state/asrh/st8090ts.txt $urls"
+urls="https://www2.census.gov/programs-surveys/popest/tables/1990-2000/state/totals/st-99-03.txt $urls"
```

Comment:
Can you load the list of URLs from a file instead of concatentating them like this?